paraoleg/lab1-BasicIO/user.h

```diff
@@ -0,0 +1,25 @@
+/******************************************************************************/
+/* User Level #define Macros                                                  */
+/******************************************************************************/
+
+// Add I/O name macros here
+#define LD1_PORT_BIT LATGbits.LATG6
+#define LD2_PORT_BIT LATDbits.LATD4
+#define LD3_PORT_BIT LATBbits.LATB11
+#define LD4_PORT_BIT LATGbits.LATG15
+#define BTN1_PORT_BIT PORTAbits.RA5
+#define BTN2_PORT_BIT PORTAbits.RA4
+/* TODO Application specific user parameters used in user.c may go here */
+
+/******************************************************************************/
+/* User Function Prototypes                                                    /
+/******************************************************************************/
+
+/* TODO User level functions prototypes (i.e. InitApp) go here */
+
+void InitApp(void);         /* I/O and Peripheral Initialization */
+
+void Flash_LED(void);
```

Comment:
When prototype is declared, the description of what function does should be located somewhere nearby. Like the line above: `void InitApp(void);         /* I/O and Peripheral Initialization */`